packages/teams-js/test/public/media.spec.ts

```diff
@@ -669,6 +517,31 @@ describe('media', () => {
           } as DOMMessageEvent);
         });
       });
+      it('should not invoke video callback for MediaControllerEvent when not registered', async () => {
+        await mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.ios);
+        mobilePlatformMock.setClientSupportedSDKVersion(nonFullScreenVideoModeAPISupportVersion);
+        let mediaError: SdkError;
+        const mediaInputs: media.MediaInputs = {
+          mediaType: media.MediaType.Video,
+          maxMediaCount: 10,
+          videoProps: {},
+        };
+        media.selectMedia(mediaInputs, (e: SdkError, attachments: media.Media[]) => {
+          mediaError = e;
+        });
+        const message = mobilePlatformMock.findMessageByFunc('selectMedia');
+        expect(message).not.toBeNull();
+        expect(message.args.length).toBe(1);
+        const callbackId = message.id;
+        mobilePlatformMock.respondToMessage({
+          data: {
+            id: callbackId,
+            args: [undefined, undefined, 2],
+          },
+        } as DOMMessageEvent);
+        expect(mediaError).toBeFalsy();
+        expect(jest.fn()).not.toHaveBeenCalled();
```

Comment:
You haven't assigned the callback insde the selectMedia call to the jest.fn(). Correct it

Reply:
oh, i think i just moved this code, i will fix it, but my next task is actually to make sure all the tests in this file are running as appropriate